src/components/Contentful/StoryCard.vue

```diff
@@ -49,17 +49,21 @@
 			v-else
 		>
 			<dynamic-rich-text
-				class="tw-text-action"
+				class="story-card__title tw-text-action"
 				:class="`tw-text-${alignment}`"
+				v-if="cardTitle"
```

Comment:
I'm a bit concerned that making these conditional will affect the layout of existing Story Cards, specifically the distribution of the flex column...

Could you please load up `/lp/own-the-change` locally to compare against https://www.dev.kiva.org/lp/own-the-change to verify.

Reply:
`/lp/eddie-test-page` is another good comparison point

Reply:
alternatively you can tests any of these pages in dev against cms page server by replacing `/lp` with `/cpslp`.

Reply:
I don't see any visual differences between dev and this PR running on localhost for both of those pages on both large and small screens.